skyportal/tests/frontend/test_newsfeed.py

```diff
@@ -100,22 +97,22 @@ def test_news_feed_prefs_widget(
         # Comment item
         driver.wait_for_xpath(f'//p[contains(text(),"comment_text_{i}")]')
 
-    driver.click_xpath('//*[@id="newsFeedSettingsIcon"]', scroll_parent=True)
```

Comment:
Safe to remove these?

Reply:
Locally it seems to work better without them.